updated docs (#913)
Graylog2/documentation: pages/plugins.rst

Maven is a widely used build tool for Java, that comes pre-installed on many operating systems or can be installed using most package managers. Make sure that you have at least version 3 before you go on.

Use it like this::
First you should install the latest version of the `graylog-project-cli <https://github.com/graylog2/graylog-project-cli/releases>`_ on your workstation. Use this to bootstrap the meta project in your working directory with the following command::

  $ git clone git@github.com:Graylog2/graylog-project.git
  $ graylog-project bootstrap github://Graylog2/graylog-project.git

This will create a checkout of the meta project in your current work dir. Now change to the ``graylog-project`` directory and follow the ``Install CLI Tool`` and ``Bootstrap`` from its README.

> Bootstrapping the ``garylog-project`` will create the directory ``graylog-project-repos``. It contains all required repositories and will also be the home of your new plugin.

Reviewer: Garylog! Hahah, glad that's getting fixed.

> This will create a checkout of the meta project in your current directory. Now have the ``graylog-project`` and ``graylog-project-repos``. Those two directories contain the meta data and all repositories that are required. The directory ``graylog-project-repos`` will also be the home of your new plugin.

Reviewer: ```suggestion
This will create a checkout of the meta project in your current directory. You'll see both ``graylog-project`` and ``graylog-project-repos``. Those two directories contain the meta data and all repositories that are required. The directory ``graylog-project-repos`` will also be the home of your new plugin.
```

At the very minimum you need to implement two interfaces:

  * ``org.graylog2.plugin.Plugin`` - which is the entry to your `plugin <https://github.com/Graylog2/graylog-plugin-sample/blob/master/src/main/java/org/graylog/plugins/sample/SamplePlugin.java>`_ code
  * ``org.graylog2.plugin.PluginMetaData`` - which `describes your plugin <https://github.com/Graylog2/graylog-plugin-sample/blob/master/src/main/java/org/graylog/plugins/sample/SampleMetaData.java>`_

The ``bootstrap-plugin`` script generates these implementations for you, and you simply need to fill out the details.

> Graylog uses Java's `ServiceLoader <https://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html>`_ mechanism to find your plugin's main class, so if you rename your ``Plugin`` implementation, you need to also adjust the `service file <https://github.com/Graylog2/graylog-plugin-sample/blob/master/src/main/resources/META-INF/services/org.graylog2.plugin.Pluginn>`_.

Reviewer: Is that a typo here?

```suggestion
Graylog uses Java's `ServiceLoader <https://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html>`_ mechanism to find your plugin's main class, so if you rename your ``Plugin`` implementation, you need to also adjust the `service file <https://github.com/Graylog2/graylog-plugin-sample/blob/master/src/main/resources/META-INF/services/org.graylog2.plugin.Plugin>`_.
```
